projects/augeas/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -eu
+#
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+./autogen.sh
+CC=clang ./configure --enable-static --disable-shared --without-selinux
+make -j$(nproc)
+
+for fuzzer in augeas_escape_name_fuzzer; do
```

Comment:
This is ok as the first step, but we recommend to have the fuzz targets be tighter integrated into the project's build system. See https://github.com/google/oss-fuzz/blob/master/docs/ideal_integration.md